.gitlab.mk

```diff
@@ -133,10 +133,10 @@ deploy:
 	echo ${GPG_SECRET_KEY} | base64 -d | gpg --batch --import || true
 	./tools/update_repo.sh -o=${OS} -d=${DIST} \
 		-b="${LIVE_REPO_S3_DIR}/${BUCKET}" build
-	if [ "${CI_COMMIT_TAG}" != "" ]; then \
+	case "${GITHUB_REF}" in refs/tags/*) \
```

Comment:
Sorry for nitpicking, but can we format / indent it in a usual way ([example](https://github.com/gentoo/gentoo/blob/e1d029fc4617996f7e2d95104b6e2661fd85e93c/net-wireless/wpa_supplicant/files/wpa_supplicant-init.d#L24-L45))?

Reply:
Sure, corrected style.